src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -731,6 +733,16 @@ You can set the [aggregation window duration](/docs/alerts-applied-intelligence/
   Baseline alert condition thresholds don't support editing the aggregation window. They use the 1 minute default.
```

Comment:
Actually, I think baselines will now have variable window support. Checking with Asia Kane to verify.

Reply:
Sean is correct, on Feb 1, baselines will support editing the aggregation window (i.e. static conditions and baseline conditions should have parity for aggregation window)